docs/modules/ROOT/pages/reference/extensions/disruptor.adoc

```diff
@@ -28,3 +28,8 @@ Please refer to the above links for usage and configuration details.
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Camel Quarkus limitations
+
+The disruptor-vm provide support for communication across CamelContext instances but since camel-quarkus support a single CamelContext, such component is not available.
```

Comment:
```suggestion
The `disruptor-vm` component is not available on Camel Quarkus. This is because it is supposed to provide support for communication across multiple CamelContext instances within a single JVM, but by design, there is always just a single `CamelContext` on Camel Quarkus. Therefore `disruptor-vm` would make no sense.
```